### DISCUSSION TITLE
kubebuilder support on windows platform.

### DISCUSSION THREAD

**Discussion by vireshnavalli** [kubernetes-sigs/kubebuilder#3330] [Support] [answered]
Hi Team, 
I see issues related to kubebuilder support on windows platform.

Does Kubebuilder compatible with windows. If yes can anyone share installation guides for the same.

**Comment by camilamacedo86** [comment, marked as answer]
This information is described in the README see: https://github.com/kubernetes-sigs/kubebuilder#supportability

![Screenshot 2023-09-16 at 13 50 42](https://github.com/kubernetes-sigs/kubebuilder/assets/7708031/bc637255-25cb-4f63-a4f1-301fd546c210)

Therefore, windows is not officially supported.
However, we might able to make it work by using solutions like  https://www.cygwin.com/ 

We would like to support windows and we would accepted contributions towards this initiative. We just do not have the effort to work on.  We have an issue open for that: https://github.com/kubernetes-sigs/kubebuilder/issues/2940. Please feel free to contribute with

I am closing this one since seems answered. 
However, feel free to re-open if you wish.
